R/load_forecasts_repo.R

```diff
@@ -0,0 +1,95 @@
+#' Load all available forecasts submitted on forecast_dates from local hub repo.
+#' 
+#' The function will throw an error when no forecasts are submitted on 
+#' any dates in forecast_dates for selected models, locations, types and target.
+#' 
```

Comment:
Add documentation for file_path parameter
```suggestion
#' 
#' @param file_path path to the data-processed folder within a local clone of the hub repo
```